PublicMapping/districtbuilder
# Fix maps page slowness

## Overview

Both the Home screen and the Community Maps screen saw a big hit to performance after adding the mini-maps to them.
In addition, the Community Maps screen was accidentally querying for projects twice, exacerbating the issue.

Since we don't need the `districts` column to be extremely detailed for the mini-maps, we can simplify it server side to greatly reduce the amount of data sent (I saw a nearly 100x improvement in size) at the cost of increased application server time / CPU load.

### Checklist


### Notes

This needs the changes to how duplicate projects are handled in #892 or it would cause Project districts to be simplified every-time they are duplicated - obviously not good.

## Testing Instructions

- Ensure you have a good number of projects visible on your Home screen & the Community Maps screen
- On this branch, time to load both pages should be greatly reduced
- You should observe only one call to the ``globalProjects` endpoint in the Network tab of the Developer tools when the page is loaded

Closes #885
